client/src/components/ProductDetails/StyleSelector.jsx

```diff
@@ -1,41 +1,30 @@
+/* eslint-disable react/jsx-key */
 /* eslint-disable react/prop-types */
 import React from 'react';
 import './prodStyles.css';
 
-//can get rid of - simply get from
-function StyleSelector ({productStylesArray, currentStyleID, currentProductID, changeCurrentStyle}) {
-  // const getCurrentProductObject = (targetId, allProducts) => {
-  //   return allProducts.filter((product) => {
-  //     return product.product_id === targetId;
-  //   });
-  // };
+function StyleSelector ({productStylesArray, currentStyleID, changeCurrentStyle}) {
 
-  // const currentProductObject = getCurrentProductObject(currentProductID, productStylesArray);
 
   const getCurrentStyleObject = (targetStyleId, stylesArray) => {
     return stylesArray.filter((style) => {
       return style.style_id === targetStyleId;
     });
   };
 
-  // const stylesArray = currentProductObject[0]['results'];
-
   const currentStyleObject = getCurrentStyleObject(currentStyleID, productStylesArray);
 
-
   return (
-    <div clasName="container">
+    <div className="flex-container wrap">
       <h2>{currentStyleObject[0]['name']}</h2>
```

Comment:
this is confusing me. Is currentStyleObject and object or an array?

Reply:
It is confusing, my apologies. Since I used the filter method to get this object, it created an array with only one element, which is the Object. Referring to it as currentStyleObject makes sense, but maybe only to me. I will rename